nautobot/ui/src/components/Layout.js

```diff
@@ -65,7 +58,7 @@ export default function Layout({ children }) {
                         to="/"
                     />
                 </Heading>
-                {sessionInfo && sessionInfo.logged_in && <SidebarNav />}
+                {isLoggedIn && <SidebarNav />}
```

Comment:
Should the sidebar also be hidden if not logged in to make the login view as generic as possible (#3732)

Reply:
Not blocking for this PR just a thought

Reply:
When we re-do the login view, yes, but for now it's where the legacy UI toggle lives.